_layouts/page.html

```diff
@@ -28,6 +28,9 @@
 				<!-- Right Nav Section -->
 				<ul class="right">
 					<li><a href="/">Home</a></li>
+					<li>
+						<a href="/culture.html"></a>
```

Comment:
Yep. And has a self merge -- `git fetch origin; git rebase origin/master` will probably straighten that out.